packages/service-library/src/servicelib/redis.py

```diff
@@ -26,6 +26,12 @@
 _DEFAULT_SOCKET_TIMEOUT: Final[datetime.timedelta] = datetime.timedelta(seconds=30)
 
 
+_DEFAULT_DECODE_RESPONSES: Final[bool] = True
+_DEFAULT_HEALTH_CHECK_INTERVAL: Final[datetime.timedelta] = datetime.timedelta(
```

Comment:
Q: I wonder why do you use a background task to check the healthcheck instead of simply calling it when you run `health`?  
With this approach, `.health` will return True if redis was healthy at latest `heath_check_interval` seconds ago

IMO this approach is more suitable in cases where you need trigger an event when some condition happens